src/index.d.ts

```diff
@@ -497,6 +497,7 @@ declare module '@sendbird/uikit-react/ChannelList' {
 declare module '@sendbird/uikit-react/ChannelList/context' {
   export type ChannelListProvider = React.FunctionComponent<ChannelListProviderProps>;
   export function useChannelListContext (): ChannelListProviderInterface;
```

Comment:
Don't need to remove the previous hook? or are we going to do it all in one time